tests/load-tests.php

```diff
@@ -148,6 +150,52 @@ public function test_perflab_render_generator() {
 		$this->assertContains( $expected, $output );
 	}
 
+	public function test_perflab_get_valid_modules() {
+		global $wp_version;
+
+		// Assert that it returns the empty array for dummy modules.
+		$dummy_active_modules = array(
+			'../tests/testdata/demo-modules/javascript/demo-module-1',
+			'../tests/testdata/demo-modules/something/demo-module-2',
+			'../tests/testdata/demo-modules/images/demo-module-3',
+		);
+
+		$output = perflab_get_valid_modules( $dummy_active_modules );
+		$this->assertIsArray( $output );
+		$this->assertSame( array( '../tests/testdata/demo-modules/something/demo-module-2', '../tests/testdata/demo-modules/images/demo-module-3' ), $output );
+
+		// Assert that it returns the array for modules.
+		if ( $wp_version >= '6.1' ) {
+			$active_modules = require plugin_dir_path( PERFLAB_MAIN_FILE ) . 'default-enabled-modules.php';
+			add_filter(
+				'perflab_active_modules',
+				function() use ( $active_modules ) {
+					return $active_modules;
+				}
+			);
+			$output = perflab_get_valid_modules( perflab_get_active_modules() );
+
+			// Remove Image module as it will marge in 6.1.
+			$remove_marge_modules = array_shift( $active_modules );
+			$this->assertSame( $active_modules, $output );
+		}
```

Comment:
I think this would be better to remove. This is not a test for the infrastructure here, it's a test for whether the WebP Uploads module-specific logic in its `can-load.php` file behaves as expected.

So either we should migrate this test into a very simple test as part of the WebP Uploads module (only `require` the file and check that the closure function does the right thing) or remove it. It shouldn't be part of the infrastructure tests, since that file is not part of the infrastructure, but of the WebP Uploads module.